integration-cli/docker_cli_images_test.go

```diff
@@ -58,42 +51,29 @@ func (s *DockerSuite) TestImagesOrderedByCreationDate(c *check.C) {
 	id1, err := buildImage("order:test_a",
 		`FROM scratch
 		MAINTAINER dockerio1`, true)
-	if err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(err, checker.IsNil)
 	time.Sleep(1 * time.Second)
 	id2, err := buildImage("order:test_c",
 		`FROM scratch
 		MAINTAINER dockerio2`, true)
-	if err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(err, checker.IsNil)
 	time.Sleep(1 * time.Second)
 	id3, err := buildImage("order:test_b",
 		`FROM scratch
 		MAINTAINER dockerio3`, true)
-	if err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(err, checker.IsNil)
 
 	out, _ := dockerCmd(c, "images", "-q", "--no-trunc")
 	imgs := strings.Split(out, "\n")
-	if imgs[0] != id3 {
-		c.Fatalf("First image must be %s, got %s", id3, imgs[0])
-	}
-	if imgs[1] != id2 {
-		c.Fatalf("Second image must be %s, got %s", id2, imgs[1])
-	}
-	if imgs[2] != id1 {
-		c.Fatalf("Third image must be %s, got %s", id1, imgs[2])
-	}
+	c.Assert(imgs[0], checker.Equals, id3, check.Commentf("First image must be %s, got %s", id3, imgs[0]))
+	c.Assert(imgs[1], checker.Equals, id2, check.Commentf("First image must be %s, got %s", id2, imgs[1]))
+	c.Assert(imgs[2], checker.Equals, id1, check.Commentf("First image must be %s, got %s", id1, imgs[2]))
 }
 
 func (s *DockerSuite) TestImagesErrorWithInvalidFilterNameTest(c *check.C) {
 	out, _, err := dockerCmdWithError("images", "-f", "FOO=123")
-	if err == nil || !strings.Contains(out, "Invalid filter") {
-		c.Fatalf("error should occur when listing images with invalid filter name FOO, %s", out)
-	}
+	c.Assert(err, checker.NotNil, check.Commentf("Error should occur when listing images with invalid filter name FOO, %s", out))
+	c.Assert(out, checker.Contains, "Invalid filter", check.Commentf("Error should indicate there was an invalid filter, %s", out))
```

Comment:
Just a nit, but no need for `out` here since it will already be printed by the assert failure.